src/content/manifest/hybrid-development.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Hybrid development
+description: Hybrid development allows you to develop a service on-premises and run a service on-premises while the other components of your app run in the cloud.
+sidebar_label: Hybrid development
+id: hybrid-development
+---
+
+import Image from '@theme/Image';
+
+Okteto offers two different approaches to service development: sync ot hybrid mode. The command affected by this is the [`okteto up`](/reference/cli/#up) command but it is at the manifest level where this feature is configured.
+
+The default mode used is the *sync* mode in which changes made to the local service are synchronized with the remote development container deployed in the cluster. On the other hand, the *hybrid* mode allows to run the development service locally and the rest of the application components in the cloud.
+
+### Problem to solve
+
+Sometimes running the service locally is faster than the original approach taken in okteto where the service runs in the cloud and changes are synchronized from local to the remote development container. Another possibility can be if you’re using a framework that is IDE-heavy like on Java/Spring, IDEs tend to be pre-configured to run tests, start services, and connect debuggers. You lose some of that ‘click and you are done’ when you  run your development service in the cluster. This is where hybrid development comes in.
+
+If you want to adopt this approach, you need to keep in mind that developing with the service running locally breaks the replicability between OSs in addition to the added latency when accessing the rest of the services in the cluster. 
+
+### How hybrid development works?
+Usually, okteto allows you to develop a service locally and run it in a production-like cloud environment synchronizing the local changes with the development container running in the cloud (sync mode) but okteto also allows you to use another approach in which the service to be developed runs locally and the rest of the application components run in the cloud (hybrid mode).
+
```

Comment:
@rlamana should we include an image about the arch here? similar to https://github.com/okteto/okteto/blob/master/docs/okteto-architecture.png

Reply:
If I create a diagram, I think we should also add a description of the how, to me this paragraph talks more about what it is rather that how it is achieved...

Reply:
I found a diagram we already had something like this:

<img width="561" alt="Screenshot 2023-12-14 at 15 39 12" src="https://github.com/okteto/docs/assets/237819/3a990298-de59-4c82-a485-d3206ab0a5fe">

Reply:
@rlamana could we also have the image representing the state before `okteto up`? and in this image, I would replace the `api` text inside the namespace by `SSH Proxy`

Reply:
@pchico83 What about something like this:

<img width="1070" alt="Screenshot 2023-12-15 at 12 01 07" src="https://github.com/okteto/docs/assets/237819/1b5db928-2609-446c-bff7-55e54efa9bd0">

Reply:
I've added the last one ATM

Reply:
@pchico83 can you review the diagram?